tslint.json

```diff
@@ -28,5 +21,6 @@
 		"strict-type-predicates": false,
 		// We need private class properties to start with an underscore.
 		"variable-name": [true, "allow-leading-underscore"]
-	}
+	},
+	"linterOptions": { "exclude": ["framework"] }
```

Comment:
why do we need this option?

Reply:
If we run tslint command in the root level, we can ignore framework folder with that option.